packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -2115,15 +2114,14 @@ describe("SharedTree", () => {
 		view.initialize([]);
 		assert.throws(
 			() => {
-				Tree.runTransaction(view, () => {
+				view.runTransaction(() => {
 					tree.connect({
 						deltaConnection: runtime.createDeltaConnection(),
 						objectStorage: new MockStorage(),
 					});
 				});
 			},
-			(e: Error) =>
-				validateAssertionError(e, /Cannot attach while a transaction is in progress/),
+			validateUsageError(/^Cannot attach while a transaction is in progress/),
```

Comment:
This used to get an "Invalid use of Shared Tree after it was put into an invalid state by another error.\nOriginal Error:\nError: Cannot attach while a transaction is in progress. Commit or abort the transaction before attaching." error. Now it validates the error more specifically (checks the message starts with this string, and that its a UssageError)